### PR TITLE
Rollbar again

### PR DESCRIPTION
**Pull request by Sybrand**
@tahitihat my previous fix actually just broke another thing (really need to get linting in on this repo too!)

### CODE REVIEW

File: config/settings.py [hunk 1]
```diff
@@ -4,3 +4,4 @@
 ROLLBAR_ACCESS_TOKEN = (
     os.getenv('ROLLBAR_ACCESS_TOKEN', None) if IS_PRODUCTION and not IS_TEST else None
 )
+ENABLE_ROLLBAR = ROLLBAR_ACCESS_TOKEN is not None
```
Comment (Sybrand) [inline config/settings.py:7]: @tahitihat this feels a bit like a snake eating it's head - but I guess we do need both. You can't configure rollbar without the token, and you we need a way to know if rollbar is enabled or not. (unless there's a way to interrogate it)

**Review by tahitihat — APPROVED** [review]
Looks great! I'm sorry I missed the problem with your last PR — can you share more about the issue it introduced, for future reference?

**Comment by Sybrand** [comment]
> 

I had removed the code that set `ENABLE_ROLLBAR` to `False`/`True` - so at that point in the code, it would just have evaluated to `False` because `ENABLE_ROLLBAR` would always be `None`

**Comment by Sybrand** [comment]
> I had removed the code that set `ENABLE_ROLLBAR` to `False`/`True` - so at that point in the code, it would just have evaluated to `False` because `ENABLE_ROLLBAR` would always be `None`

I had removed the code that set ENABLE_ROLLBAR to False/True - so at that point in the code, it would just have evaluated to False because ENABLE_ROLLBAR would always be None

**Pull request merged**